### PR TITLE
feature: result formatter extensibility with minor refactor and sever…

### PR DESCRIPTION
**Pull request by dsalahutdinov**
…al testsy

Hi. Some refactor to make adding new result view format depend of content-type easy

### CODE REVIEW

**Comment by asciimoo** [comment]
@dsalahutdinov great pull request, thanks. #57 consolidates lots of things in the code, so it would be good to merge that before this PR, if you agree.

**Comment by dsalahutdinov** [comment]
@asciimoo , that's ok, i will renew my PR after merging of yours

**Comment by asciimoo** [comment]
@dsalahutdinov thanks.
#59 (reopened #57) merged, please rebase.

**Comment by dsalahutdinov** [comment]
@asciimoo i've done
Please take a look at .travis.yml, it has been set to run tests instead of build binaries

File: .travis.yml [hunk 1]
```diff
@@ -9,9 +9,7 @@ os:
 env:
   - "PATH=/home/travis/gopath/bin:$PATH"
 script:
-  - test -z "$(gofmt -l ./)"
-  - test -z "$(go vet -v ./...)"
-  - go build
```
Comment (asciimoo) [inline .travis.yml]: I think, these checks would be still useful. Is `go test` evaluates these too?

Comment (Benaiah) [inline .travis.yml]: Travis *should* be running tests, IMO. The `gofmt` check in particular has already caught a couple misformatted PRs.

Comment (Benaiah) [inline .travis.yml]: Yeah, unless `go test` already does it, the `gofmt` check should definitely remain there. It's already caught a couple misformatted PRs.

**Comment by asciimoo** [comment]
+1, except for travis mods. @dsalahutdinov could you please add `go fmt` to travis and rebase it to the current master?

**Comment by dsalahutdinov** [comment]
Yes. I'll do it soon. Having some holidays now

**Comment by dsalahutdinov** [comment]
@asciimoo , i've done

File: wuzz.go [hunk 2]
```diff
@@ -668,28 +661,14 @@ func (a *App) PrintBody(g *gocui.Gui) {
...
-			(ctype == CONTENT_TYPES["json"] || strings.HasSuffix(ctype, "+json")) {
-			formatter := jsoncolor.NewFormatter()
-			buf := bytes.NewBuffer(make([]byte, 0, len(req.RawResponseBody)))
-			err := formatter.Format(buf, req.RawResponseBody)
-			if err == nil {
-				responseBody = buf.Bytes()
-			}
-		}
+		responseFormatter := formatter.New(a.config, req.ContentType)
+		vrb.Title = "Response body (F9) " + responseFormatter.Title()
```
Comment (asciimoo) [inline wuzz.go]: Could you use https://github.com/asciimoo/wuzz/blob/master/wuzz.go#L191 instead of hard-coding the title prefix?

Comment (dsalahutdinov) [inline wuzz.go]: done, sorry for my inattention :disappointed:

**Comment by asciimoo** [comment]
Thank you

**Pull request merged**